Makefile

```diff
@@ -131,8 +131,10 @@ docker:
 	@git archive --format=tar.gz HEAD | docker build -t hypothesis/hypothesis:$(DOCKER_TAG) -
 
 .PHONY: docker-ws
-docker-ws:
+docker-ws: docker
+	docker tag hypothesis/hypothesis:dev hypothesis/hypothesis:latest 
 	docker build -t hypothesis/hypothesis-ws:$(DOCKER_TAG) h/streamer 
+	docker rmi hypothesis/hypothesis:latest
```

Comment:
I don't think you want to remove the `hypothesis/hypothesis` image. It might have been built by the user themselves with `make docker` for use by `make run-docker`.

Reply:
This removes the tag not the image. `hypothesis/hypothesis:dev` will remain.